llvm/test/MC/X86/cmpccxadd-intel-64.s

```diff
@@ -768,3 +768,6 @@
 // CHECK: encoding: [0xc4,0x62,0xa9,0xe4,0x8a,0x00,0xfc,0xff,0xff]
```

Comment:
Why was it encoded correctly before this PR? I think r9 needs R bit too.

Reply:
// CHECK: cmpsxadd  %r10, %r9, 268435456(%rbp,%r14,8)
It happens `%r10 ` `wrongly` set the R bit. So I added a test of `%ecx, %r8d, (%rip)`